components/dropdown/dropdown-content-mixin.js

```diff
@@ -316,6 +317,7 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		this.mediaQueryList = window.matchMedia(`(max-width: ${this.mobileBreakpointOverride - 1}px)`);
 		this._useMobileStyling = this.mediaQueryList.matches;
 		if (this.mediaQueryList.addEventListener) this.mediaQueryList.addEventListener('change', this._handleMobileResize);
+		if (this.opened) this.__addRepositionHandlers();
```

Comment:
We only really want the reposition handlers wired up while the dropdown is open. These `__addRepositionHandlers` and `__removeRepositionHandlers` in the connected callbacks serve to handle the cases where the dropdown is added/removed to/from the DOM while open.